### DISCUSSION TITLE
스페셜한 Spatial AI 러너 참여 이메일 오발송

### DISCUSSION THREAD

**Discussion by changh95** [Pseudo-Lab/9th-builder#30] [Issue and Q&A] [answered]
참가 이메일을 보내주시면서 문제가 있었던 것 같습니다. 
러너분들 중 한분께서 (김선호님) 탈락 이메일과 디스코드 초대를 동시에 받으셨다고 합니다.
현재 리포트 받은건 없지만, 혹시 다른 러너분들께서도 동일한 현상을 겪으시면서 혹시나 선정 안되신건줄 알고 참여를 못하실 수도 있지 않을까 걱정이 됩니다.
번거로우시겠지만 확인 해주시면 좋을 것 같습니다.

감사합니다!

**Comment by seriousran** [comment, marked as answer]
디스코드에서 답변드렸지만, 기록을 위해 깃헙에서도 답변드립니다.

스페셜한 spatial ai 멤버들에게는 모두 탈락 이메일이 발송되었었습니다... 
어디선가 누락이 된 듯 합니다... 죄송합니다! 
해당하는 8명에게 선정 메일을 다시 발송하였습니다..!
